app/pages/home-for-user/my-builds.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import HomePageSection from './generic-section';
+import { Link } from 'react-router';
+import ProjectCard from '../../partials/project-card';
+
+const NOOP = Function.prototype;
+
+const MyBuildsSection = React.createClass({
+  propTypes: {
+    onClose: React.PropTypes.func.isRequired,
+  },
+
+  contextTypes: {
+    user: React.PropTypes.object.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      loading: false,
+      error: null,
+      projects: [],
+      avatars: {},
+    };
+  },
+
+  getDefaultProps() {
+    return {
+      onClose: NOOP,
```

Comment:
I think using an arrow function, `() => {}` will be cleaner, clearer. Also consistent with our other JSX components.